core/src/com/unciv/logic/simulation/Simulation.kt

```diff
@@ -94,6 +109,16 @@ class Simulation(
         println("Simulation step ($stepCounter/$maxSimulations)")
     }
 
+    @Synchronized
+    fun saveStat(gameInfo: GameInfo) {
+        val turn = gameInfo.turns
+        for (civ in gameInfo.civilizations.filter { it.civName != Constants.spectator }) {
+            val popsum = civ.cities.sumOf { it.population.population }
+            //println("$civ $popsum")
+            sumStat[civ.civName]!![turn]!!.set(sumStat[civ.civName]!![turn]!!.get() + popsum)
```

Comment:
doesn't MutableInt have an add() function?

Reply:
Nope! Just inc()

Reply:
Now it does